config/formats.ts

```diff
@@ -1697,22 +1665,16 @@ export const Formats: FormatList = [
 
 		mod: 'gen8',
 		team: 'randomCC',
-		teamLength: {
-			battle: 1,
-		},
-		ruleset: ['[Gen 8] Challenge Cup', 'Team Preview', 'Dynamax Clause'],
+		ruleset: ['[Gen 8] Challenge Cup', 'Team Preview', 'Dynamax Clause', 'Chosen Team Size = 1'],
 	},
 	{
 		name: "[Gen 8] Challenge Cup 2v2",
 
 		mod: 'gen8',
 		team: 'randomCC',
 		gameType: 'doubles',
-		teamLength: {
-			battle: 2,
-		},
 		searchShow: false,
-		ruleset: ['[Gen 8] Challenge Cup 1v1'],
+		ruleset: ['[Gen 8] Challenge Cup 1v1', '!Chosen Team Size', 'Chosen Team Size = 2'],
```

Comment:
I like the explicitness. It might be more convenient to auto-override, but the manual override is a useful reminder in case you didn't know you were overriding an earlier rule.

Reply:
One thing we might want to do is to consider a syntax to intentionally repeal and replace a rule in one command.

Like `Chosen Team Size := 2`

Reply:
It's unfortunate that I'd want to use `!=` or `==` but those already have meanings in programming languages that would be very weird to use here.

Reply:
Or: `Chosen Team Size = 1 -> 2`

For literally the most explicitness possible.

Reply:
(That last one unfortunately doesn't work well for arbitrary string values, though.)

Reply:
Genius idea: `Chosen Team Size = 2 != 1`

Reply:
Ended up going with `!! Chosen Team Size = 2`